code/modules/martial_arts/bearserk.dm

```diff
@@ -0,0 +1,75 @@
+/datum/martial_art/bearserk
+	weight = 9 // Higher weight than Krav-Maga, brute force overrules
+	name = "Rage of the Space Bear"
+	has_explaination_verb = TRUE
+	combos = list(/datum/martial_combo/bearserk/bear_jaws, /datum/martial_combo/bearserk/paw_slam, /datum/martial_combo/bearserk/smokey)
+
+/datum/martial_art/bearserk/harm_act(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	MARTIAL_ARTS_ACT_CHECK
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	var/atk_verb = pick("punches", "claws", "hits", "mauls")
+	playsound(get_turf(D), 'sound/weapons/punch1.ogg', 25, TRUE, -1)
+	D.visible_message("<span class='danger'>[A] [atk_verb] [D]!</span>", "<span class='userdanger'>[A] [atk_verb] you!</span>")
+	D.apply_damage(10, BRUTE, A.zone_selected)
+	if(D.stat != DEAD)
+		A.adjustStaminaLoss(-10)
```

Comment:
Adjusting the staminaloss on someone dead isnt really going to affect them, I recommend removing this check

Reply:
That's supposed to be a check to prevent recovering stamina when hitting a dead person

Reply:
But after doublechecking other code, it should be (isLiving) instead, so I'll do that